app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  validates :first_name, :last_name, presence: { message: 'Please, fill the field!' },
+                                     format: { with: /\A[A-Z].*\z/, message: 'Please, start from upper case' }
+  validates :email, uniqueness: true
```

Comment:
please, add validations for email and phone. Use regepx for validate input and add presence true